### PR TITLE
Updated .htaccess to include ieee-1451.0-schema

### PR DESCRIPTION
**Pull request by rbrama79**
Dear Josh, I hope to have correctly updated the file in order to link to ieee-1451.0-schema repository. Hope this helps you. Best regards.

### CODE REVIEW

File: ieee/.htaccess [hunk 1]
```diff
@@ -80,6 +80,36 @@ RewriteRule ^ieee-1752-schema/physical-activity\.json$ https://opensource.ieee.o
...
+RewriteRule ^ieee-1451.0-schema/SecurityTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/SecurityTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/SmartTransducerDataModel.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/SmartTransducerDataModel.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/TextTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/TextTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/TimeSyncTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/TimeSyncTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/TransducerChannelIdentificationTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/TransducerChannelIdentificationTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/UnitsExtensionTEDS.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/UnitsExtensionTEDS.xsd [R=302,L]
+RewriteRule ^ieee-1451.0-schema/examples/E2.xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/examples/E2.xml [R=302,L]
```
Comment (davidlehn) [inline ieee/.htaccess:97]: If these patterns will stay the same, all these main rules and examples rules could be replaced by a single pattern with optional `examples` dir:
```suggestion
RewriteRule ^ieee-1451.0-schema/((examples/)?\w+).xsd$ https://opensource.ieee.org/1451/1451.0-schema/-/blob/main/normative/$1.xsd [R=302,L]
```

**Comment by davidlehn** [comment]
cc: @joshgay

**Comment by joshuagay** [comment]
@davidlehn I agree. I will make a PR from my orgs account with this suggested change. 
@rbrama79 Thank you!

**Comment by davidlehn** [comment]
New PR is here: https://github.com/perma-id/w3id.org/pull/3263

**Comment by davidlehn** [comment]
Applied similar rules in https://github.com/perma-id/w3id.org/pull/3263.

**Pull request closed without merging**